barretenberg/cpp/src/barretenberg/vm2/tracegen/lib/instruction_spec.cpp

```diff
@@ -76,8 +76,9 @@ const std::unordered_map<ExecutionOpCode, SubtraceInfo> SUBTRACE_INFO_MAP = {
       { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = AVM_EXEC_OP_ID_RETURNDATASIZE } },
     { ExecutionOpCode::DEBUGLOG,
       { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = AVM_EXEC_OP_ID_DEBUGLOG } },
-    // KeccakF1600
+    // Hashes
     { ExecutionOpCode::KECCAKF1600, { .subtrace_selector = SubtraceSel::KECCAKF1600, .subtrace_operation_id = 0 } },
+    { ExecutionOpCode::POSEIDON2PERM, { .subtrace_selector = SubtraceSel::POSEIDON2PERM, .subtrace_operation_id = 0 } },
```

Comment:
Not a big deal but is this really poseidon2perm _trace_? It's actually the opcode/mem one I guess? I don't mind if you keep it as is, I think it's what we are doing with others.